src/thorlabs_cube/aqctl_thorlabs_cube.py

```diff
@@ -59,10 +60,12 @@ def main():
                 dev = TpzSim()
             elif product == "kdc101":
                 dev = KdcSim()
+            elif product == "kpz101":
+                dev = KpzSim()
             else:
                 print(
```

Comment:
I think this should be a raised `ValueError`, rather than a print and then exit.